compute/ARMComputeEx/arm_compute/core/NEON/kernels/NEBinaryLogicalOperationKernel.h

```diff
@@ -41,15 +41,20 @@
 #ifndef __ARM_COMPUTE_NEBINARYLOGICALOPERATIONKERNEL_H__
 #define __ARM_COMPUTE_NEBINARYLOGICALOPERATIONKERNEL_H__
 
-#include "arm_compute/core/NEON/kernels/NEElementwiseOperationKernel.h"
 #include "arm_compute/core/TypesEx.h"
 
+#include "src/core/cpu/kernels/CpuElementwiseKernel.h"
+
 namespace arm_compute
 {
 
-class NEBinaryLogicalOperationKernel : public NEElementwiseOperationKernel
+class NEBinaryLogicalOperationKernel
+  : public cpu::kernels::CpuComparisonKernel // CpuComparisonKernel
```

Comment:
I think this one looks better
`class NEBinaryLogicalOperationKernel : public cpu::kernels::CpuComparisonKernel`

`// CpuComparisonKernel` is not necessary because you already wrote `cpu::kernels::CpuComparisonKernel`

Reply:
@Hyunjun85 Please check this.

Reply:
Thanks, I have updated PR.